### PR TITLE
Assert on the path requested when accessing well known attr

### PR DESCRIPTION
**Pull request by alexsnaps**
Makes sure you are actually accessing the path you expect... 
Most useful to document [these](https://github.com/Kuadrant/wasm-shim/pull/129/files#diff-030f3ad71048fac9886241d6c33bcd3741a027e0403b506e115f5a6c5e37dcb7R477-R482)

### CODE REVIEW

File: src/data/cel.rs [hunk 1]
```diff
@@ -470,36 +472,65 @@ mod tests {
...
         let value = Expression::new("auth.identity.age").unwrap().eval();
+        assert_eq!(value, "John".into());
+
+        property::test::TEST_PROPERTY_VALUE.set(Some((
+            property::Path::new(vec!["filter_state", "wasm.kuadrant.auth.identity.name"]),
+            "-42".bytes().collect(),
+        )));
+        let value = Expression::new("auth.identity.name").unwrap().eval();
         assert_eq!(value, (-42).into());
+
```
Comment (adam-cattermole) [inline src/data/cel.rs:512]: I guess it doesn't really matter as such as the path is somewhat irrelevant to the change but minor nit; age == John, name == -42

**Review by adam-cattermole — APPROVED** [review]
lgtm

File: src/data/cel.rs (hunk 1 above)
Comment (alexsnaps) [reply src/data/cel.rs:512]: I actually did it on purpose, to _not_ imply typing based of what otherwise could

Comment (adam-cattermole) [reply src/data/cel.rs:512]: Ah, sure!

**Review by didierofrivia — APPROVED** [review]
🧑‍🚀

**Pull request merged**